zoom/users.py

```diff
@@ -220,7 +220,7 @@ def can_run(self, app_name):
         """test if user can run an app"""
         return self.is_admin or self.is_developer or app_name in self.apps
 
-    def can(self, action, thing):
+    def can(self, action, thing=None):
```

Comment:
confirmed, you can undo this change at this line - I symlinked in the recent apps and this issue disappeared